src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -354,6 +354,7 @@ export default function MissingFileOverview({
               data={sponsorMiss}
               nameKey="name"
               title={oh}
+              subset={false}
```

Comment:
Rather than not passing a value for `subset` (implicit false), pass an explicit value. This was all that was needed to fix it.